app/src/main/java/edu/mills/cs115/fruitthief/map/Markers.kt

```diff
@@ -0,0 +1,41 @@
+package edu.mills.cs115.fruitthief.map
+
+import android.app.Application
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.Observer
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.google.android.gms.maps.CameraUpdateFactory
+import com.google.android.gms.maps.GoogleMap
+import com.google.android.gms.maps.model.LatLng
+import com.google.android.gms.maps.model.MarkerOptions
+import edu.mills.cs115.fruitthief.database.FruitTreeDAO
+import edu.mills.cs115.fruitthief.database.Tree
+
+class Markers () : Observer<Tree> {
+
```

Comment:
You don't need this many blank lines. There are some other formatting issues here too. `Code > Reformat Code`.